app/bg/dbs/settings.js

```diff
@@ -57,10 +63,12 @@ export const once = events.once.bind(events)
 
 /**
  * @param {string} key
- * @param {string | number} value
+ * @param {string | number | object} value
  * @returns {Promise<void>}
  */
 export async function set (key, value) {
+  if (['default_search_engines', 'selected_search_engine'].includes(key))
```

Comment:
On conditionals, please use brackets unless it's all one line, so either:

```
if (cond) dothing() // <-- only do this rarely too
```

or

```
if (cond) {
  doThings()
}
```

but never

```
if (cond)
  doThing()
```

We follow this rule to avoid accidental dangling ifs like Heartbleed suffered from.

Reply:
Makes total sense.